src/openvic-simulation/politics/Issue.cpp

```diff
@@ -73,19 +81,33 @@ bool IssueManager::add_reform(std::string_view identifier, ReformGroup const* gr
 		return false;
 	}
 
-	return reforms.add_item({ identifier, *group, ordinal });
+	if (group->get_type().is_uncivilised() != (technology_cost != 0)) {
+		Logger::warning("Technology cost ", technology_cost, " found in civilised reform ", identifier, "!");
+	}
+
+	return reforms.add_item({ identifier, std::move(values), *group, ordinal, std::move(rules), technology_cost });
 }
 
-bool IssueManager::_load_issue_group(size_t& expected_issues, std::string_view identifier, ast::NodeCPtr node) {
+bool IssueManager::_load_issue_group(size_t expected_issues, std::string_view identifier, ast::NodeCPtr node) {
 	return expect_length([&expected_issues](size_t size) -> size_t {
 		expected_issues += size;
 		return size;
 	})(node) & add_issue_group(identifier);
 }
```

Comment:
```suggestion
bool IssueManager::_load_issue_group(size_t& expected_issues, std::string_view identifier, ast::NodeCPtr node) {
	return expect_length(add_variable_callback(expected_issues))(node) & add_issue_group(identifier);
}
```
`expected_issues` needs to be a reference and the lambda function inside `expect_length` can be replaced with `add_variable_callback`.